gto/tag.py

```diff
@@ -183,16 +183,26 @@ def find(
     return tags
 
 
-def create_tag(repo, name, ref, message):
+def create_tag(
+    repo: git.Repo,
+    name: str,
+    ref: str,
+    message: str,
+    GIT_COMMITTER_NAME: str = None,
+    GIT_COMMITTER_EMAIL: str = None,
+):
     if all(c.hexsha != ref for c in repo.iter_commits()):
         raise RefNotFound(ref=ref)
     if name in repo.refs:
```

Comment:
Also not this PR, but it's better to perform a faster test first, in case it fails. Not really important for this case but still.

Reply:
Thanks for the feedback. Will keep in mind.